client/my-sites/plans-features-main/index.jsx

```diff
@@ -117,23 +117,30 @@ export class PlansFeaturesMain extends Component {
 			term = TERM_ANNUALLY;
 		}
 
+		const group = displayJetpackPlans ? GROUP_JETPACK : GROUP_WPCOM;
+
+		// In WPCOM, only the business plan is available in monthly term
+		// For any other plan, switch to annually.
+		const businessPlanTerm = term;
+		if ( group === GROUP_WPCOM && term === TERM_MONTHLY ) {
+			term = TERM_ANNUALLY;
+		}
+
 		let plans;
-		if ( displayJetpackPlans ) {
-			const group = GROUP_JETPACK;
+		if ( group === GROUP_JETPACK ) {
 			plans = [
 				findPlansKeys( { group, type: TYPE_FREE } )[ 0 ],
 				findPlansKeys( { group, term, type: TYPE_PERSONAL } )[ 0 ],
 				findPlansKeys( { group, term, type: TYPE_PREMIUM } )[ 0 ],
 				findPlansKeys( { group, term, type: TYPE_BUSINESS } )[ 0 ],
 			];
 		} else {
-			const group = GROUP_WPCOM;
 			plans = [
 				findPlansKeys( { group, type: TYPE_FREE } )[ 0 ],
 				findPlansKeys( { group, term, type: TYPE_BLOGGER } )[ 0 ],
 				findPlansKeys( { group, term, type: TYPE_PERSONAL } )[ 0 ],
 				findPlansKeys( { group, term, type: TYPE_PREMIUM } )[ 0 ],
-				findPlansKeys( { group, term, type: TYPE_BUSINESS } )[ 0 ],
+				findPlansKeys( { group, term: businessPlanTerm, type: TYPE_BUSINESS } )[ 0 ],
```

Comment:
When rendered from `signup/steps/plans/index` `intervalType` is not set and thus defaulted to annual. Is that the desired outcome?

Jetpack plans-grid looks like it is specific and I assume you're testing on the plans page. May be worthwhile adding `required` to the props definition while we're at it.

Reply:
Been a few days but [I suspect](https://github.com/Automattic/wp-calypso/blob/a1ec0c15fab52833eca64e2127596a40ba84d8df/client/my-sites/plans-features-main/index.jsx#L304): setting `.isRequired` on `intervalType`.

In other words is `term` defaulting yearly the desired outcome for signups only? - i.e. should signup flow use annual while plans page upgrades use monthly? I'm not 100% sure if this is actually the case now 25 odd days later I may have just been mentioning we should make a hard requirement for consumers to be more specific in their expectations instead of relying on 
 defaults.

Reply:
Yes, yearly is the default for signups, since the change to monthly is made on checkout either way.

Reply:
Does that complicate conversion rate comparisons?